package.json

```
@@ -6,10 +6,12 @@
     "dev": "turbo dev",
     "lint": "turbo lint",
     "clean": "rimraf -g \"**/node_modules\" \"**/dist\" \"**/.turbo\" \"**/.next\"",
-    "format": "prettier --write \"**/*.{ts,tsx,md}\""
+    "format": "prettier --write \"**/*.{ts,tsx,md}\"",
+    "release": "pnpm publish -r"
```

```suggestion
    "release-npm": "pnpm publish -r"
```

I don't like making this more verbose, but there are a bunch of different things in our monorepo including Python code, various apps, etc.. and it can be confusing what just "release" means.